OloEditor/src/Panels/DialogueEditorPanel.cpp

```
@@ -0,0 +1,1794 @@
+
+    // =========================================================================
+    // Toolbar
+    // =========================================================================
+
+    void DialogueEditorPanel::DrawToolbar()
+    {
+        if (ImGui::BeginMenuBar())
+        {
+            if (ImGui::BeginMenu("File"))
+            {
+                if (ImGui::MenuItem("New", "Ctrl+N"))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)